src/browser/builds/algoliasearchLite.js

```diff
@@ -3,4 +3,4 @@
 var AlgoliaSearchCore = require('../../AlgoliaSearchCore.js');
 var createAlgoliasearch = require('../createAlgoliasearch.js');
 
-module.exports = createAlgoliasearch(AlgoliaSearchCore, 'lite ');
+module.exports = createAlgoliasearch(AlgoliaSearchCore, 'Browser (lite)');
```

Comment:
I think we should avoid the parens here to have an easier to parse, but will leave the call up to @mthuret

Reply:
Marie agreed on the format, we can move forward with it.